docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -326,6 +326,265 @@ There you can find links to upgrade notes for other versions too.
 - follow instructions in the [separate article](/docs/upgrade/upgrade-instructions-for-read-model-for-product-lists-from-elasticsearch.md) to use Elasticsearch to get data into read model ([#1096](https://github.com/shopsys/shopsys/pull/1096))
 - follow upgrade instructions for entities simplification in the [separate article](./upgrade-instructions-for-entities-simplification.md) ([#1123](https://github.com/shopsys/shopsys/pull/1123))
 - if you want to use our experimental backend API, read [introduction to backend API](/docs/backend-api/introduction-to-backend-api.md) ([#1055](https://github.com/shopsys/shopsys/pull/1055))
+- add RabbitMQ to your production technology stack, more info can be found in [Installation on Production Server](/docs/installation/installation-using-docker-on-production-server.md) ([#1228](https://github.com/shopsys/shopsys/pull/1228))
```

Comment:
Hi, could you maybe move the instructions into a separate file? Also, why "production" technology stack? I believe it is not only for production...and in the linked article, I can not see anything about rabbit so I am a  little bit confused

Reply:
I moved instructions to separate file. 

Production documentation will be updated soon so there will be info how to install RabbitMQ and run consumers in production.